src/dsymbol/conversion/second.d

```diff
@@ -236,6 +250,7 @@ do
 	size_t i = 0;
 	foreach (part; lookup.breadcrumbs[])
 	{
+		if(part == "*") continue;
```

Comment:
```suggestion
		if (part == POINTER_SYMBOL_NAME) continue;
```

Reply:
actually this is debugging left over, this check is not needed since suffixes are removed https://github.com/dlang-community/dsymbol/blob/471b3f17248734ccf28061e919496928b3448bc1/src/dsymbol/conversion/second.d#L198